sklearn/linear_model/_ransac.py

```diff
@@ -201,7 +204,7 @@ class RANSACRegressor(MetaEstimatorMixin, RegressorMixin,
     References
     ----------
     .. [1] https://en.wikipedia.org/wiki/RANSAC
-    .. [2] https://www.sri.com/sites/default/files/publications/ransac-publication.pdf
+    .. [2] https://www.sri.com/sites/default/files/publications/ransac-publication.pdf  # noqa
```

Comment:
Are you sure it has an effect inside a docstring string?

Reply:
I put it at the end of the docstring.